.storybook/main/main.js

```diff
@@ -0,0 +1,30 @@
+let config = require('../main');
+
+config.stories.push('../../packages/pharos/**/*.docs.stories.mdx');
+
+config.refs = (config, { configType }) => {
+  if (configType === 'DEVELOPMENT') {
+    return {
+      webComponents: {
+        title: 'Web Components',
+        url: 'http://localhost:9000',
+      },
+      react: {
+        title: 'React',
+        url: 'http://localhost:9001',
+      },
+    };
+  }
+  return {
+    webComponents: {
+      title: 'Web Components',
+      url: 'https://pharos.jstor.org/storybooks/wc',
+    },
+    react: {
+      title: 'React',
+      url: 'https://pharos.jstor.org/storybooks/react',
+    },
```

Comment:
May want to end these with a slash to avoid a redirect.

Reply:
Er, maybe it won't even work that way given the redirect rules. Still a bit muddy for me there regarding the Netlify behavior.